src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/admin/AdminPageIT.java

```diff
@@ -31,6 +31,18 @@ public void accessPageAsAdmin() {
 		page.clickUsersSubMenu();
 		page.clickGroupsLink();
 		assertTrue("Admin can navigate to groups page, groups page title should be present", page.comparePageTitle("Groups"));
+		// Navigate to clients page
```

Comment:
Add tests for the individual buttons on those pages as well. Such as the `Add User` button on the Users List etc. to make sure the functionality is intact.

Reply:
I have added test for visibility of the individual buttons.